src/diffusers/pipelines/controlnet/pipeline_controlnet.py

```diff
@@ -633,7 +633,17 @@ def check_inputs(
             # When `image` is a nested list:
             # (e.g. [[canny_image_1, pose_image_1], [canny_image_2, pose_image_2]])
             elif any(isinstance(i, list) for i in image):
-                raise ValueError("A single batch of multiple conditionings is not supported at the moment.")
+                if len(image) != len(prompt):
+                    raise ValueError(
+                        f"If image batch size is not 1, image batch size must be same as prompt batch size. image batch size: {len(image)}, prompt batch size: {len(prompt)}"
```

Comment:
```suggestion
```

I think this will be checked inside `check_image()` - is it the case?  https://github.com/huggingface/diffusers/blob/79df50388df09d9615e3c067695a453bb0a694c0/src/diffusers/pipelines/controlnet/pipeline_controlnet.py#L732

Reply:
Yes, you are right about this. There's no need for a check out of `check_image()`.